rfm-master/eeprom.h

```diff
@@ -44,12 +44,12 @@
 
 #define EEPROM __attribute__((section(".eeprom")))
 
-typedef struct {                                        // each variables must be uint8_t or int8_t without exception
-#if (RFM == 1)
-	/* 00...07 */ uint8_t	security_key[8];        //!< key for encrypted radio messasges
+typedef struct                                          // each variables must be uint8_t or int8_t without exception
+{ #if (RFM == 1)

```

Comment:
Not sure this is ok. According to [GCC doc](https://gcc.gnu.org/onlinedocs/gcc-9.2.0/cpp/The-preprocessing-language.html#The-preprocessing-language), "Preprocessing directives are lines in your program that start with ‘#’. Whitespace is allowed before and after the ‘#’." So I would say the #if should be on a separate line.